SpatialGDKEditorToolbar/Source/Private/InteropCodeGenerator/SpatialGDKEditorInteropCodeGenerator.cpp

```diff
@@ -246,6 +247,7 @@ bool SpatialGDKGenerateInteropCode()
 		PlatformFile.CreateDirectory(*SchemaOutputPath);
 	}
 
+	check(GetDefault<UGeneralProjectSettings>()->bSpatialNetworking);
```

Comment:
How come we need this?

Reply:
If bSpatialNetworking is false, then schema could be incorrectly generated.

Reply:
This is a separate change though. Would be nice to move into a separate PR that can be merged now (since this one is blocked by https://github.com/improbableio/UnrealEngine/pull/23)